.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 install:
 - testing/install_test_dependencies.sh
+- echo "extension = mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
sorry, it's `services` (plural), like [here](https://github.com/bshaffer/oauth2-server-php/blob/develop/.travis.yml#L19)